src/components/Home/Card/index.scss

```
@@ -0,0 +1,60 @@
+.card__container {
```

podes usar el "&" para conectar la propiedad nueva al selector actual, entonces esto se compila en ".card__container" ahorrándote bastante tiempo :P https://css-tricks.com/snippets/sass/caching-current-selector-sass/
```suggestion
.card {
	&__container {
		display: flex;
		flex-direction: column;
		align-items: center;
		background-color: #03111f;
		color: #fcfcfc;
		padding: 15px;
		margin: 10px;
	}

	&__img {
		width: 100%;
		height: 80%;
		object-fit: cover;
		border-radius: 10px;
	}
}

```